lib/openshift/cluster_operator.rb

```diff
@@ -18,5 +18,22 @@ def related_objects(user: nil, cached: true, quiet: false)
       return rr.dig('status', 'relatedObjects')
     end
 
+    def wait_till_ready(user, seconds)
+      res = nil
+      iterations = 0
+      start_time = monotonic_seconds
+
+      success = wait_for(seconds) {
+        res = ready?(user: user, quiet: true)
+        iterations = iterations + 1
```

Comment:
`iterations` is assigned but not used

Reply:
@jhou1 good catch, let's make it simple and removed it now.